src/pages/Contact.js

```diff
@@ -0,0 +1,14 @@
+import { GridContainer } from '@trussworks/react-uswds';
+import Form from '../components/ContactForm';
+
+function TeamPage() {
```

Comment:
Did you maybe mean to write `function Contact() {` rather than `function TeamPage()` as you already have a TeamPage component in your Team.js file?